test/prime_db_schema_client.py

```diff
@@ -0,0 +1,54 @@
+import uuid
+from layman import settings, app
+from layman.common.prime_db_schema import publications, workspaces, users
+
+latest_oauth2_sub = 0
+
+
+def ensure_workspace(workspace):
+    with app.app_context():
+        return workspaces.ensure_workspace(workspace)
+
+
+def ensure_user(workspace):
+    workspace_id = ensure_workspace(workspace)
+    global latest_oauth2_sub
+    latest_oauth2_sub += 1
+    user_info = {
+        'sub': latest_oauth2_sub,
+        'issuer_id': 'layman',
+        'claims': {
+            'email': f"{workspace}@liferay.com",
+            'name': workspace,
+            'middle_name': '',
+            'family_name': workspace,
+            'given_name': workspace,
+            'preferred_username': workspace,
+        }
+    }
+    with app.app_context():
+        users.ensure_user(workspace_id, user_info)
+
+
+def post_workspace_publication(publication_type, workspace, name, actor, access_rights=None, title=None, bbox=None):
+    access_rights = access_rights or {}
+    for right_type in ['read', 'write']:
+        access_rights[right_type] = access_rights.get(right_type, {settings.RIGHTS_EVERYONE_ROLE})
```

Comment:
Make this the same as in rest API. Actor, when specified, EVERYONE for anonymous user.